src/main/server/enable_ssl.sh

```diff
@@ -157,11 +157,24 @@ if [ "$chainFileExist" != "$privateKeyFileExist" ]; then
 fi
 
 get_freedomain(){
+
+  check_marketplace() {
+    if curl -s http://169.254.169.254/latest/meta-data/ >/dev/null; then
+      return 0
+    elif curl -H Metadata:true -s http://169.254.169.254/metadata/instance?api-version=2021-02-01 >/dev/null; then
```

Comment:
Could you just mention that AWS and GCP uses the same private ip address as a comment to not have any misunderstanding?